safe/impact_function/test/test_impact_function.py

```diff
@@ -384,9 +386,12 @@ def test_scenarii(self):
             'raster_continuous_on_raster_population': False,
         }
 
+        # If we want to invert the selection.
+        invert = True
+
         path = standard_data_path('scenario')
         for scenario, enabled in scenarii.iteritems():
```

Comment:
Hah, I thought scenarii is plural also. It should be scenari anyway in Italian. Reference http://english.stackexchange.com/questions/11775/what-is-the-plural-of-scenario